### ISSUE TITLE
Link in forward messages

### ISSUE THREAD

**Issue by ahmaddxb** [micl2e2/tg-focus#19]
The links are not handled correctly in the forwarded messages

![image](https://github.com/user-attachments/assets/28243209-9a19-45ce-b880-414042cafa8f)

Orginal message
![image](https://github.com/user-attachments/assets/cde10840-363e-4284-9e2a-6ac4f4718334)

**Comment by ahmaddxb** [comment]
Is there a fix for this?

**Comment by ahmaddxb** [comment]
I tired testing this today but seem to be the same, has this been worked on?

**Comment by micl2e2** [comment]
Yes, as long as the issue being actively tagged, and not being closed, it is considered to-be-resolved. Actually the project is undergoing a significant refactoring. Will got notified once all the necessary work done.
